package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ts-common/json-parser",
-  "version": "0.3.1",
+  "version": "0.4.0",
```

Comment:
would this break only oav ? I dont think we would use directly in the services ?

Reply:
I use the library in several other different places so I need to bump the version.